Cython/TestUtils.py

```diff
@@ -228,6 +232,17 @@ def unpack_source_tree(tree_file, workdir, cython_root):
     return workdir, header
 
 
+def fresh_cythonize(*args, language_level=3, **kwargs):
```

Comment:
Should we make the default language level 3? I read that cython wants to drop support for python 2.7 by 3.1 release.

Reply:
Yes, no, at least now it will not work, so I [remove `anguage_level`](https://github.com/cython/cython/pull/4241/commits/21438d198f2f9e2fd47a3d2e8f5b2200ccab27f5).